ext/gd/tests/gh10614.phpt

```diff
@@ -5,6 +5,9 @@ gd
 --SKIPIF--
 <?php
 if (!GD_BUNDLED && version_compare(GD_VERSION, '2.3.4', '>=')) die("skip test requires GD 2.3.4 or older");
```

Comment:
Otherwise offtopic but this also looks strange in this test. This test is for GD >= 2.3.4 or for GD <= 2.3.4?

Reply:
It passes for me with external libgd-2.3.3, so I'm assuming that it's looking for the old (buggy) behavior that was fixed in the bundled libgd and will be fixed in upstream libgd-2.3.4. But from the expected output I can't really be sure. Ask @nielsdos :)

Reply:
Yeah, I submitted the patch and it was merged (https://github.com/libgd/libgd/pull/862) but a new release of libgd is not made yet. It is fixed in the bundled version indeed.
I don't remember the details of this check. I would expect this test to fail with external libgd 2.3.3 tbh.

Reply:
It passes.

Reply:
Ok, I didn't understand this test. Of course that's fine. It must be summer heat here 😂